library/src/main/java/com/chuckerteam/chucker/internal/support/AndroidCacheFileFactory.kt

```diff
@@ -10,7 +10,14 @@ internal class AndroidCacheFileFactory(
     private val fileDir = context.cacheDir
     private val uniqueIdGenerator = AtomicLong()
 
-    override fun create(): File {
-        return File(fileDir, "chucker-${uniqueIdGenerator.getAndIncrement()}")
+    override fun create(): File = File(fileDir, "chucker-${uniqueIdGenerator.getAndIncrement()}")
```

Comment:
If you are adding `createExportFile` please provide some meaningful name for this fun as well.

Reply:
updated